admin/admin-service/src/main/java/io/mosip/admin/service/impl/AdminServiceImpl.java

```diff
@@ -73,5 +106,93 @@ private void createLostRidRequest(SearchInfo searchInfoRequest) {
 
 	}
 
+	public String getBiometric(byte[] isodata) throws Exception {
+		    ConvertRequestDto convertRequestDto = new ConvertRequestDto();
+		    convertRequestDto.setVersion("ISO19794_5_2011");
+			convertRequestDto.setInputBytes(isodata);
+			byte[] data = FaceDecoder.convertFaceISOToImageBytes(convertRequestDto);
+			String encodedBytes = new String(data);
+
+		return encodedBytes;
+	}
+
+	@Override
+	public LostRidDetailsDto lostRidDetails(String rid) {
+		LostRidDetailsDto lostRidDetailsDto=new LostRidDetailsDto();
+		Map<String,String> lostRidDataMap=new HashMap<>();
+		FieldDtos fieldDtos=new FieldDtos();
+		RequestWrapper<FieldDtos> fieldDtosRequestWrapper=new RequestWrapper<>();
+		ConvertRequestDto convertRequestDto = new ConvertRequestDto();
+		try {
+			FieldResponseDto fieldResponseDto=new FieldResponseDto();
+			buildSearchFieldsRequestDto(fieldDtos,rid);
+			fieldDtosRequestWrapper.setRequest(fieldDtos);
+			ResponseWrapper<FieldDtos> fieldDtosResponseWrapper = restClient.postApi(ApiName.PACKET_MANAGER_SEARCHFIELDS, MediaType.APPLICATION_JSON,
+					fieldDtosRequestWrapper, ResponseWrapper.class);
+			fieldResponseDto = objectMapper.readValue(objectMapper.writeValueAsString(fieldDtosResponseWrapper.getResponse()), FieldResponseDto.class);
+			FieldResponseDto finalFieldResponseDto = fieldResponseDto;
+			for (String field: fields) {
+				if (finalFieldResponseDto.getFields().containsKey(field) && field.equalsIgnoreCase(nameField)) {
+					String value = finalFieldResponseDto.getFields().get(field);
+					org.json.JSONArray jsonArray = new org.json.JSONArray(value);
+					org.json.JSONObject jsonObject = (org.json.JSONObject) jsonArray.get(0);
+					lostRidDataMap.put(field, jsonObject.getString(VALUE));
+				} else {
+					lostRidDataMap.put(field, finalFieldResponseDto.getFields().get(field));
+				}
+			}
+			getBiometric(rid,lostRidDataMap);
+			lostRidDetailsDto.setLostRidDataMap(lostRidDataMap);
+		} catch (Exception e) {
+			throw new RequestException(LostRidErrorCode.UNABLE_TO_RETRIEVE_LOSTRID_DATA.getErrorCode(),
+					LostRidErrorCode.UNABLE_TO_RETRIEVE_LOSTRID_DATA.getErrorMessage()
+							+ e);
+		}
+		return lostRidDetailsDto;
+	}
 
+	private void getBiometric(String rid, Map<String, String> lostRidDataMap){
```

Comment:
change the method name to getApplicantPhoto

Reply:
taken care